### PR TITLE
Adapt NPM scripts to be OS-agnostic

### PR DESCRIPTION
**Pull request by BillFarber**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,20 +58,10 @@ cd server
...
-    * [Migration Instructions](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/migrating-from-windows-powershell-51-to-powershell-7?view=powershell-7.2)
-    * Note: this is currently broken because of [an open bug in npm](https://github.com/npm/cli/issues/5332).  Use Bash.  :(
-        
-You can choose which environment should be used as the default shell for npm with the command `npm config set script-shell [shell]`.  For instance, to use WSL Bash, you would run `npm config set script-shell bash`.  PowerShell v7+ is `npm config set script-shell pwsh`  You can reset this with `npm config delete script-shell`.  Even if running npm from PowerShell, it will try to use cmd.exe as the default shell if you do not define it.
-
-To compile changes, run `npm run-script compile`.  Note that both the client and the language server need to finish compiling before you attempt to run the extension, or you will get an error.  Other script definitions are in `package.json.`
-
+A shorthand script that does all of that for you is
+```
+npm run npmInstallClientAndServer
```
Comment (rjrudin) [inline CONTRIBUTING.md:63]: 🌶️

**Review by rjrudin — APPROVED** [review]
LGTM, though I don't have a way of verifying this on Windows.

**Review by bshalke — APPROVED** [review]
I'm off today so I didn't set up ML to run the integration tests, but the client and server tests both work with no failures for me now under Windows with the cmd shell.

Thanks for taking care of this so quickly.

**Comment by BillFarber** [comment]
Thanks for testing this out, Bryan.

**Pull request merged**